android/src/com/mapswithme/maps/editor/data/NamesDataSource.java

```diff
@@ -0,0 +1,27 @@
+package com.mapswithme.maps.editor.data;
+
+/**
+ * Class which contains array of loclized names with following priority:
+ *  1.Names for Mwm languages
```

Comment:
This languages are retrieved from Java_com_mapswithme_maps_editor_Editor_nativeGetNamesDataSource